unlock-app/src/components/creator/CreatorLock.js

```diff
@@ -29,7 +30,7 @@ export class CreatorLock extends React.Component {
     // TODO add USD values to lock
     // TODO add all-time balance to lock
 
-    const { lock, transaction, config } = this.props
+    const { lock, transaction, keys, config } = this.props
```

Comment:
I see you still pass keys here but they're not needed here. I was thinking you could extract them in the `KeyList` component altogether!